tools/accuracy_checker/accuracy_checker/preprocessor/geometric_transformations.py

```diff
@@ -480,18 +480,31 @@ def parameters(cls):
         parameters.update({
             'src_landmarks': ListField(
                 description='Source landmark points',
-                value_type=ListField(value_type=int)
+                value_type=ListField(value_type=float)
             ),
             'dst_landmarks': ListField(
                 description='Destination landmark points',
-                value_type=ListField(value_type=int)
+                value_type=ListField(value_type=float)
+            ),
+            'dst_height': NumberField(
+                description='Destination height size',
+                value_type=int,
+                optional=False
+            ),
+            'dst_width': NumberField(
+                description='Destination width size',
+                value_type=int,
+                optional=False
             )
         })
         return parameters
 
     def configure(self):
         self.src_landmarks = self.get_value_from_config('src_landmarks')
         self.dst_landmarks = self.get_value_from_config('dst_landmarks')
+        self.dst_height = self.get_value_from_config('dst_height')
```

Comment:
@hojaedev do you plan to update existing configs or stay backward compatibility (if parameters are not provided, source height and width will be used)?

Reply:
I will update the configuration for the VAS algorithm that uses warp affine: face recognition.

Reply:
ok, please do not forget to update